plugins/obs-webrtc/whip-output.cpp

```diff
@@ -164,12 +216,35 @@ bool WHIPOutput::Setup()
 		obs_output_signal_stop(output, OBS_OUTPUT_BAD_PATH);
 		return false;
 	}
+
 	bearer_token = obs_service_get_connect_info(
 		service, OBS_SERVICE_CONNECT_INFO_BEARER_TOKEN);
 
+	return true;
+}
+
+/**
+ * @brief Set up the PeerConnection and media tracks.
+ * 
+ * @return true 
+ * @return false 
```

Comment:
Please remove trailing spaces in the lines.

Reply:
@norihiro is this something to watch out for, I assumed clang-format handled this scenario?